### DISCUSSION TITLE
Error from NetCDF in build of ufs-weather-model for NG-GODAS (-NEMSDATM)

### DISCUSSION THREAD

**Discussion by hyunchul386** [ufs-community/ufs-weather-model#657] [Q&A] [answered]
I would like to ask about an error of NetCDF in Cmake;
In building for the app of NG-GODAS-NEMSDATM, at Orion, there occurred as following error,

CMake Error at CMakeLists.txt:108 (find_package):
  By not providing "FindNetCDF.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "NetCDF", but
  CMake did not find one.

  Could not find a package configuration file provided by "NetCDF" with any
  of the following names:

    NetCDFConfig.cmake
    netcdf-config.cmake
---------
Current modules are
Currently Loaded Modules:
  1) contrib/0.1     6) intel/2018.4      11) jasper/2.0.22  16) pio/2.5.2       21) ip/3.3.3     26) nemsio/2.5.2  31) ufs_orion.intel
  2) noaatools/1.0   7) mkl/2020.2        12) zlib/1.2.11    17) esmf/8_1_1      22) sp/2.3.3     27) w3emc/2.7.3
  3) cmake/3.18.1    8) hpc-intel/2018.4  13) png/1.6.35     18) fms/2020.04.03  23) sigio/2.3.2  28) crtm/2.3.0
  4) python/3.7.5    9) impi/2018.4       14) hdf5/1.10.6    19) g2/3.4.2        24) bacio/2.4.1  29) upp/10.0.6
  5) hpc/1.1.0      10) hpc-impi/2018.4   15) netcdf/4.7.4   20) g2tmpl/1.10.0   25) w3nco/2.4.1  30) ufs_common

I checked this in branch of develop 
Do I miss something? How can I fix it?

Thanks

**Comment by DusanJovic-NOAA** [comment, marked as answer]
This works for me on Orion:

```
git clone --recursive https://github.com/ufs-community/ufs-weather-model
cd ufs-weather-model
module use modulefiles
module load ufs_orion.intel
CMAKE_FLAGS="-DAPP=NG-GODAS-NEMSDATM" ./build.sh
```

**Comment by hyunchul386** [comment]
Thanks Dusan             -- Hyun-Chul

**Reply by junwang-noaa** [reply]
@hyunchul386 Is your issue resolved?

**Comment by hyunchul386** [comment]
Yes, it was resolved.

Thanks,
Hyun-Chul
